Development-Guide/Git-Guides-and-Standards/Commit-Messages.adoc

```diff
@@ -21,6 +21,8 @@ Further paragraphs come after blank lines.
   single space. Use a hanging indent.
 ```
 
+Your commit title should be phrased so it can answer the question "If this commit is applied, it will..."
+
 ### Example commit message
```

Comment:
Where is the example message?

Reply:
Isn't it on line 29? I didn't change that, so it doesn't show up in the diff